src/store/modules/game.js

```diff
@@ -237,10 +238,11 @@ export default {
       state.waitingForOpponentToPlayFromDeck = val;
     },
     // Game Over
-    setGameOver(state, { gameOver, conceded, winner }) {
+    setGameOver(state, { gameOver, conceded, winner, currentMatch }) {
       state.gameIsOver = gameOver;
       state.conceded = conceded;
       state.winnerPNum = winner;
+      state.currentMatch = currentMatch;
```

Comment:
I think we also need to set this in `resetState()` so it initializes correctly (do we want `null`?